.luacheckrc

```diff
@@ -69,6 +69,7 @@ read_globals = {
 	'GetUnitPowerBarInfo',
 	'GetUnitPowerBarInfoByID',
 	'GetUnitPowerBarStringsByID',
+	'HasLFGRestrictions',
```

Comment:
Is this wrongly indented or is it the damn GitHub mobile app?

Reply:
![image](https://user-images.githubusercontent.com/2725970/131453100-6f73feed-c09a-4828-895d-df4821e68028.png)
All tabs.